mivideo/mivideo_extract.py

```diff
@@ -295,7 +295,7 @@ def _makeCourseData(resultDictionaries: Sequence[Dict], categoryFilter: str) ->
             c.endswith('>InContext') for c in courseData['course_id']]
 
         courseData['course_id'] = courseData['course_id'].str.replace(
-            r'^' + categoryFilter + r'.*>([0-9]+).*$',
+            r'^' + categoryFilter + r'.*>([0-9]+)(>InContext)?$',
```

Comment:
The Regex is looking for exact text `>InContext` is it possible that other combinations like `incontext` or `Incontext` are allowed?

Reply:
Can you also add some comments about what exactly you are i is a brief reason for that? Or as simple as a sample example as below should be good as well?
`Canvas_UMich>site>channels>7db438071375c02373713c12c73869ff2f470b68>InContext`

Reply:
> The Regex is looking for exact text `>InContext` is it possible that other combinations like `incontext` or `Incontext` are allowed?

No, it will be exactly `>InContext` or not at all.